src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -747,6 +747,7 @@ class DocBaseViewer extends BaseViewer {
         const { file, location } = this.options;
         const assetUrlCreator = createAssetUrlCreator(location);
         const hasDownload = checkPermission(file, PERMISSION_DOWNLOAD);
+        const hasForms = hasDownload && this.getViewerOption('disableForms') === false;
```

Comment:
Does this need a test? Also does this warrant an update to the preview sdk readme?

Reply:
Hm, I think I'll just set it to false.